css/style.min.css

```diff
@@ -1 +1,9 @@
+ "."
+ "full-width"}#main-gateway .main-gateway-full-width{-ms-grid-row:3;-ms-grid-column:1;grid-area:full-width}#main-gateway section{background:#f5f5f5;padding:1.375rem 1.25rem}#front-page-news{padding:0}#front-page-news .main-news{position:relative;background:#000;overflow:hidden}#front-page-news .main-news div{position:relative;z-index:2;color:#fff}#front-page-news .main-news div p:last-child{margin-bottom:0}#front-page-news .main-news img{position:absolute;top:0;left:0;width:100%;height:100%;-o-object-fit:cover;object-fit:cover;opacity:0.5;z-index:1}#front-page-news .more-news{display:-ms-grid;display:grid;row-gap:2rem}#front-page-news .more-news .lvl2-news{background:#f5f5f5;padding:1.375rem 1.25rem}@media only screen and (min-width: 992px){#main-gateway{-ms-grid-columns:(1fr)[2];grid-template-columns:repeat(2, 1fr);-webkit-column-gap:2rem;column-gap:2rem;row-gap:2rem;grid-template-areas:". ."
+ "full-width full-width"}#main-option-sections{-ms-grid-columns:(1fr)[5];grid-template-columns:repeat(5, 1fr);-webkit-column-gap:0.5rem;column-gap:0.5rem;grid-template-areas:". . . . ."}#front-page-news .more-news{-ms-grid-columns:(1fr)[2];grid-template-columns:repeat(2, 1fr);-webkit-column-gap:2rem;column-gap:2rem;grid-template-areas:". ."}}.bg-yellow{background-color:yellow;text-align:center}.lrt-head{font-size:40px;padding:50px, 10px}.container-box{background-color:rgba(160,159,159,0.6);border-bottom-left-radius:10px;border-bottom-right-radius:10px;-webkit-box-shadow:11px 14px 10px 0px rgba(0,0,0,0.75);box-shadow:11px 14px 10px 0px rgba(0,0,0,0.75)}.container-box>p{text-align:center;font-size:23px;font-weight:500;padding:10px}@media only screen and (min-width: 440px){.lrt-head{font-size:60px}}
```

Comment:
Near the end here is the faulty CSS, you can search for _.lrt-head{font-size:40px;padding:50px, 10px}_ to find it.
**Should be:** .lrt-head{font-size:40px;padding:50px 10px}